VIDEO_5_SCRIPT.md

```diff
@@ -0,0 +1,71 @@
+
+## Why AsyncAPI is committed to an Open Governance model
+
+Partnering with Postman was a crucial moment for assuring the efforts of the AsyncAPI Initiative. This event was the propellant to accelerate the transfer of AsyncAPI to an open government foundation to assure the community that a single company does not control the AsyncAPI Initiative. In doing so, the specification safeguarded against private interests.
+
+In the AsyncAPI governance model, you can find two essential roles: contributor and committer. A contributor is a person that contributes to the project code, docs, or other artifacts. A committer (widely known also as maintainer) is a person that contributes regularly and is invited by other committers to manage the repository, to have more privileges, and to approve pull requests.
+
+To supervise the initiative, it was decided to create a Technical Steering Committee (TSC), made up of the people who contribute most actively to the project. Anyone with interest can become a member of the TSC. All you have to do is become a maintainer of one of the AsyncAPI projects. To become a maintainer, it is necessary to regularly contribute to one of the projects; after that, other maintainers will invite them to join. 
+
+The TSC is constituted by maintainers because the goal is to empower "working" individuals, not companies that "pay". In other words, it provides equal power to individual and corporate contributors.
+
+The TSC's system of operation is by voting. An issue or debate is posted, and the members vote asynchronously and publicly on their decision. The value of the vote of all contributors is equal, thus ensuring the project that there is no risk of one company having a majority of the voters. That is why there is a rule that only 1/4 of the voters can be affiliated with the same company.
+
+## Putting theory into practice
+
+Summing up, AsyncAPI is committed to democratic management based on two principles: horizontality in decision making and transparency in project management. From the beginning of the specification, AsyncAPI has relied on collective effort and group identity. The means are as crucial as the goal. Responsibilities are shared, understanding that a community can only achieve success jointly.
+
+For doing so, AsyncAPI has several tools that ensure transparency within the project’s open-governance management. Let's see how theory is put into practice:
+
+- **Economy**: all the financial management of the project is managed through Open Collective. It’s an online crowdfunding platform for communities that provides the necessary tools to raise and share the finances raised with complete transparency. You can find out where the money comes from in this space and its use.
```

Comment:
**AsyncAPI uses the Open Collective to manage its finances. Open Collective is a crowdfunding platform that provides tools to raise and share finances with complete transparency. You can consult a money source and how it is used within a project.**